samples/copy-from-container/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns = "http://maven.apache.org/POM/4.0.0"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation = "http://maven.apache.org/POM/4.0.0
+   http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <parent>
+      <groupId>io.fabric8.dmp.samples</groupId>
+      <artifactId>dmp-sample-parent</artifactId>
+      <version>0.34-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+   <modelVersion>4.0.0</modelVersion>
+   <artifactId>dmp-sample-copy-from-container</artifactId>
+   <profiles>
+      <profile>
+         <id>Copy-File</id>
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>io.fabric8</groupId>
+                  <artifactId>docker-maven-plugin</artifactId>
+                  <executions>
+                     <execution>
+                        <id>copy-from-container</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                           <goal>start</goal>
+                           <goal>copy</goal>
+                           <goal>stop</goal>
+                        </goals>
+                        <configuration>
+                           <images>
+                              <image>
+                                 <name>busybox:latest</name>
+                                 <copy>
+                                    <entries>
+                                       <entry>
+                                          <containerPath>/etc/hosts</containerPath>
+                                          <hostDirectory>/tmp/</hostDirectory>
+                                       </entry>
+                                    </entries>
+                                 </copy>
+                              </image>
+                           </images>
+                        </configuration>
+                     </execution>
+                  </executions>
+               </plugin>
+            </plugins>
+         </build>
+      </profile>
+      <profile>
+         <id>Copy-Directory</id>
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>io.fabric8</groupId>
+                  <artifactId>docker-maven-plugin</artifactId>
+                  <executions>
+                     <execution>
+                        <id>copy-from-container</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                           <goal>start</goal>
```

Comment:
This profile could use `<createContainers>true</createContainers>`, i.e. could demonstrate copying from temporary (created, but not started) container instead of copying from the started container (like it's done in `Copy-File` profile).